src/Adapters.Tests/Etw.Net451.Tests/Etw.Net451.Tests.csproj

```diff
@@ -0,0 +1,67 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{15FA58D1-F397-4808-831C-7AE65EE640AF}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Microsoft.ApplicationInsights.EtwCollector.Tests</RootNamespace>
+    <AssemblyName>Microsoft.ApplicationInsights.EtwCollector.Net451.Tests</AssemblyName>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
+    <FileAlignment>512</FileAlignment>
+    <ProjectTypeGuids>{3AC096D0-A1C2-E12C-1390-A8335801FDAB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">10.0</VisualStudioVersion>
```

Comment:
I think you are over-concerning project file properties being broken. If Visual Studio is not handling the properties it adds to the project, it is a visual studio bug.

That said, I will remove it as I am not seeing a real problem either to remove the property.

On the side, what the code say is: If there is no variable of VisualStudioVersion specified, it will set it to 10.0 as a default value, which I guess is a backward compatibility safer. :-)